doc/Programming-guide/Key-platform-features/Common-Type-Library/index.md

```diff
@@ -9,23 +9,24 @@ sort_idx: 20
 * TOC
 {:toc}
 
-The *Common Type Library (CTL)* is a repository of reusable data type schemas that can be referenced and reused as the user creates specific schema definitions for Kaa modules.
-The *Common Type (CT)* is a unit of CTL which represents set of *data type schema versions (CT schemas)*.
-This capability unifies management of all schemas in a Kaa server instance.
+The [Common type library (CTL)]({{root_url}}Glossary/#common-type-library-ctl) is a repository of data type schemas used for all Kaa modules.
+As more schema types and versions are created, they are recorded in the CTL for future use.
 
-# CT types definitions and FQNs
+[Common type (CT)]({{root_url}}Glossary/#common-type-ct) is a CTL unit representing a set of data type schema versions.
+Using CTL allows for consistent schema management within a [Kaa instance]({{root_url}}Glossary/#kaa-instance-kaa-deployment).
 
```

Comment:
I think it's worth mentioning already here that Kaa schemas are based on Apache Avro, so that references to Avro further down the page are more clear.